pubtools/_quay/signer_wrapper.py

```diff
@@ -239,10 +239,9 @@ def _run_store_signed(self, signed_results: Dict[str, Any]) -> None:
         Signatures are uploaded in batches.
 
         Args:
-            claim_messages ([dict]):
-                Signature claim messages constructed for the RADAS service.
-            signature_messages ([dict]):
-                Messages from RADAS containing image signatures.
+            signed_results: (Dict[str, Any]):
+                Dictionary of {"operation":..., "operation_results":..., "signing_key":...}"}
```

Comment:
https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/msgsigner.py#L462
The entrypoint returns signer_result, operation_results and signing_key.

Reply:
yeah, let me fix that